provider/resource_keycloak_oidc_identity_provider.go

```diff
@@ -123,6 +123,8 @@ func getOidcIdentityProviderFromData(data *schema.ResourceData) (*keycloak.Ident
 
 	if data.HasChange("client_secret") {
 		rec.Config.ClientSecret = data.Get("client_secret").(string)
+	} else {
+		rec.Config.ClientSecret = "**********"
```

Comment:
Is this going to send the string `**********` to Keycloak on update?  I'm wondering if it would be better to remove the `if` statement on line 124 and always call `rec.Config.ClientSecret = data.Get("client_secret").(string)` so Keycloak is always getting the correct client secret on every call.

Reply:
This is how actually browser is working

Reply:
Anyway I've updated it